templates/taxi/car_detail.html

```diff
@@ -10,6 +10,14 @@
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    <form action="" method="post">
+      {% csrf_token %}
+      {% if user in car.drivers.all %}
+        <input type="submit" class="btn btn-danger link-to-page" value="Delete me from this car">
+      {% else %}
+        <input type="submit" class="btn btn-success link-to-page" value="Assign me to this car">
+      {% endif %}
+    </form>
```

Comment:
Code Style issue: Consider adding a `Cancel` button to the form. This will allow users to return to the previous page they were on if they decide not to submit the form.